bindings/wasm/identity_wasm/package.json

```diff
@@ -14,6 +14,7 @@
   },
   "scripts": {
     "build:src": "cargo build --lib --release --target wasm32-unknown-unknown --target-dir ../target",
+    "build:src:nodejs": "cargo build --lib --release --target wasm32-unknown-unknown --target-dir ../target --features keytool",
```

Comment:
Should this be used in `build:nodejs` as it was before the last `feat/identity-rebased-alpha` merge?